### DISCUSSION TITLE
bouton "home" fait planter le lunii

### DISCUSSION THREAD

**Discussion by viccho** [marian-m12l/studio#191] [Q&A]
Bonjour. Tout d'abord, merci pour votre travail, c'est vraiment super.
J'essaie de mettre des podcasts en mp3 sur la lunii. Je commence avec le noeud de départ que je lie directement à un noeud histoire(dans lequel je mets mon podcast). Tout fonctionne à merveille, sauf qu'une fois l'histoire commencée, si j'appuie sur le bouton home, ça fait planter la lunii. il faut éteindre et rallumer pour recommencer. Avez-vous un conseil ?
![question](https://user-images.githubusercontent.com/82603307/115038275-a9da3180-9ecf-11eb-9666-fdb5334cc5c2.jpg)

merci
bonne journée

**Comment by DJEX93** [comment]
Je me permet de reposter le lien de "hteumeuleu" qui a preconiser des chose a ce propos 

> 
_J’avais fait un exemple simple avec juste un noeud de départ et directement un noeud d’histoire. Mais apparemment, ça « casse » le fonctionnement du bouton retour de la Lunii (obligeant de l’éteindre et la rallumer pour revenir au choix de pack d’histoire). Je n’ai plus eu ce problème en rajoutant un noeud de menu intermédiaire._

https://web.archive.org/web/20210101121455/https://www.hteumeuleu.fr/mettre-ses-propres-histoires-sur-la-lunii/#more-17866

**Comment by marian-m12l** [comment]
@viccho @DJEX93 Effectivement c'est une limitation des noeuds "histoire", il me semble que je l'ai noté dans la partie "aide" de l'éditeur (sur la gauche), mais je ne sais pas si c'est bien visible. En revanche il est possible d'avoir le comportement souhaité en utilisant les noeuds "experts" stage/action.
